src/liblzma/common/stream_encoder_mt.c

```diff
@@ -743,7 +743,7 @@ stream_encode_mt(void *coder_ptr, const lzma_allocator *allocator,
 
 		// These are for wait_for_work().
 		bool has_blocked = false;
-		mythread_condtime wait_abs;
+		mythread_condtime wait_abs = {};
```

Comment:
Similar to the other comment, include this as a separate commit from the CI changes. 

ts funny that it flags this as uninitialized, but not on line 978 in same file. Maybe since its in a switch-case? No change needed to the other use of `mythread_condtime wait_abs;`, just an observation.